playwright-e2e/tests/singular/enrollment/self-enrollment.spec.ts

```diff
@@ -0,0 +1,166 @@
+import MyDashboardPage, { WHO } from 'tests/singular/dashboard/my-dashboard-page';
+import * as user from 'tests/singular/mock-data/fake-user.json';
+import { clickSignMeUp, goToPath } from 'tests/singular/lib/nav';
+import { makeEmailAlias, makeRandomNum } from 'tests/lib/utils';
+import { fillEmailPassword, fillSitePassword } from 'tests/lib/auth-singular';
+
+test.describe('Adult', () => {
+  test.beforeEach(async ({ page }) => {
+    await goToPath(page, '/password');
+    await fillSitePassword(page);
+    await new HomePage(page).waitForReady();
+  });
+
+  /**
+   * Test case: https://docs.google.com/document/d/1Ewsh4ULh5LVdZiUapvG-PyI2kL3XzVf4seeLq8Mt-B0/edit?usp=sharing
+   */
+  test('can finish self-enrollment @enrollment @singular', async ({ page }) => {
+    // Assertion helper functions
+    const assertActivityHeader = async (page: Page, expectedText: string) => {
+      await expect(page.locator('h1.activity-header')).toHaveText(expectedText);
+    };
+
+    const assertActivityProgress = async (page: Page, expectedText: string) => {
+      await expect(page.locator('h3.progress-title')).toHaveText(expectedText);
+    };
+
+    await clickSignMeUp(page);
+
+    // On “Create your account” page
+    const aboutYourself = new AboutYourselfPage(page);
+    await aboutYourself.waitForReady();
+    // Enter an age >= 21 and <= 99
+    await aboutYourself.age().textInput().fill(makeRandomNum(21, 99).toString());
+    await aboutYourself.country().select().selectOption(user.patient.country.abbreviation);
+    await aboutYourself.state().select().selectOption(user.patient.state.abbreviation);
+    // In the “Do you or your immediate family member have a single ventricle heart defect?” select “Yes”
+    await aboutYourself.haveVentricleHeartDefect().checkbox('Yes').check();
+    // Checkbox "I'm not a robot"
+    await aboutYourself.checkReCaptcha();
+    await aboutYourself.signMeUp.click();
+
+    // Enter email alias and new password in Login popup
+    await fillEmailPassword(page, {
+      email: makeEmailAlias(process.env.singularUserEmail as string),
+      password: process.env.singularUserPassword,
+      waitForNavigation: true
+    });
+
+    // On "My Dashboard" page
+    const myDashboardPage = new MyDashboardPage(page);
+    await myDashboardPage.waitForReady();
+    await myDashboardPage.enrollMyself.click();
+
+    // On "Enroll myself" page
+    await assertActivityHeader(page, 'Enroll myself');
+    await myDashboardPage.whoHasVentricleHeartDefect(WHO.Me).check();
+    await myDashboardPage.next.click();
+
+    // On "Consent Form" page
+    const consentForm = new ConsentFormPage(page);
+    await consentForm.waitForReady();
+    await assertActivityHeader(page, 'Your Consent Form');
+    await assertActivityProgress(page, 'Page 1 of 3');
+    await consentForm.next.click();
+    await assertActivityProgress(page, 'Page 2 of 3');
+    await consentForm.next.click();
```

Comment:
next method no longer exists as part of consent-form-page.ts as well as some other methods :

![image](https://user-images.githubusercontent.com/109761417/194152017-ea5dadfa-dcbd-416c-8881-70209b293ea4.png)

Reply:
Curious, can you find the "missing" functions in your project? If they're present, then it's got to be an IDE setting.

Reply:
I think it was something with vscode . I re-opened it and all the issues were fixed.